Gemfile

```diff
@@ -189,6 +189,9 @@ group :development do
     gem('mailcatcher', '~> 0.6.0') unless rails5? # https://github.com/sj26/mailcatcher#bundler
   gem('quiet_assets', '~> 1.1.0') unless rails5?
   gem 'launchy', '~> 2.4.0'
-  gem 'rubocop', '~> 0.60.0'
   gem 'web-console', '~> 2.3.0', '< 3.0.0'
+
+  install_if -> { '2.2.0' <= RUBY_VERSION } do
```

Comment:
Just a minor drive by comment – I've never seen `install_if` so don't know about approach – but the conditional took a bit of thinking. A more human phrasing would be `install_if -> { RUBY_VERSION >= '2.2.0' } do`.

Reply:
Yeah I agree, this does read like a yoda condition. Generally tho I always try to have conditions smallest to greatest but I guess in this case it doesn't really make much sense. I'll change it.